thirdparty/ffs/ffs/ffs/tests/adios2_bug.c

```diff
@@ -5,7 +5,6 @@
 #include <string.h>
 #include <sys/stat.h>
 
-#include "adios2/common/ADIOSConfig.h"
```

Comment:
Looks like I made a mistake here yesterday... I updated this file in ADIOS even though it's a third-party library. This line created a merge conflict when I updated from upstream. Removing the line should be the correct course of action to resolve the conflict.

Reply:
You can also `rm -rf thirdparty/ffs/ffs`, commit, and then re-run the update script and it will pull a fresh copy ignoring the in-tree history.

Reply:
Done